applications/tari_console_wallet/src/lib.rs

```diff
@@ -135,6 +135,12 @@ pub fn run_wallet_with_cli(runtime: Runtime, config: &mut ApplicationConfig, cli
         );
     }
 
+    let on_init = match boot_mode {
+        WalletBoot::New => true,
+        _ => false,
+    };
+    let on_recovery = recovery_seed.is_none();
```

Comment:
This variable seems badly named. I'm not sure if it's supposed to be `not_recovery`?

Reply:
`not_recovery` conveys the role of the variable much better